data/css/all-aboard.css

```diff
@@ -284,3 +321,15 @@ a:focus {
     height: 0;
     opacity: 0;
 }
+
+/* Hide visually, but not to screen readers */
+.visually-hidden {
```

Comment:
This is used to hide the form label elements from visual users but, it will still be accessible to screen reader users.